src/main/java/com/groupbyinc/quickstart/model/QuickstartResults.java

```diff
@@ -0,0 +1,36 @@
+package com.groupbyinc.quickstart.model;

```

Comment:
Could you create a ticket marked as technical debt to add matchStrategy to the actual results object?